tests/plugins/test_plugin_load.py

```diff
@@ -29,6 +29,9 @@ def plugin_configuration(classname):
     plugin_conf[namespace][klass]["api_key"] = "fake"
     if category == "probes":
         plugin_conf[namespace][klass]["generations"] = random.randint(2, 12)
+        # CustomPrompts requires primary_detector
+        if classname == "probes.generic.CustomPrompts":
+            plugin_conf[namespace][klass]["primary_detector"] = "always.Pass"
```

Comment:
see comment re: hardcoding in `test_probes_base` - can a solution be found that doesn't rely on a static switch in the tests?